Python/cp/Median_of_two_sorted_arrays.py

```diff
@@ -0,0 +1,71 @@
+
+# Given two sorted arrays nums1 and nums2 of size m and n
+# respectively, return the median of the two sorted arrays.  
+
+# function to find median of two arrays
+def findMedianSortedArrays(nums1, nums2):
+    newlist = sorted(nums1 + nums2)
+
+    # checking if length of merged array is even
+    if len(newlist) % 2 != 1:
+        first = newlist[len(newlist) // 2]
+        second = newlist[len(newlist) // 2 - 1]
+        return (first + second) / 2
+
+    # checking if length of merged array is odd
+    if len(newlist) % 2 == 1:
+        return newlist[len(newlist) // 2]
+
+# Driver's code
+if __name__ == "__main__":
+    # creating an empty lists
+    nums1 = []
+    nums2 = []
+    n = int(input('Enter length of first array: '))
+    m = int(input('Enter length of second array: '))
+
+    print('Enter the elements of first array: ')
+    for i in range(n):
+        x = int(input())
+        nums1.append(x)
+
+    print('Enter the elements of second array: ')
+    for i in range(m):
+        y = int(input())
+        nums2.append(y)
```

Comment:
Use list map instead of a for loop

Reply:
@rohanbabbar04 can help me? I am not getting how to change.

Reply:
@Amit366 refer this https://www.geeksforgeeks.org/python-get-a-list-as-input-from-user/ 
Read code number 3

Reply:
@rohanbabbar04 done